### PR TITLE
Development 2.6 update node price discount popup

### PR DESCRIPTION
**Pull request by amiraabouhadid**
### Changes
Updated node price and discount popup to include packages and price before/after discount

### Related Issues

#2805 

### Documentation PR

For UI changes, Please provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

### Checklist

- [x] Screenshots/Video attached (needed for UI changes)

image
image

### CODE REVIEW

**Review by zaelgohary — COMMENTED** [review]
Build fails.

**Review by Mahmoud-Emad — APPROVED** [review]
Amazing ya Amiraaa

File: packages/playground/src/components/node_selector/TfNodeDetailsCard.vue [hunk 1]
```diff
@@ -466,7 +503,35 @@ export default {
       }
     }
 
+    const discountPackagesItems = [
+      { name: "Default", discount: "-20%", tfts: 200 },
+      { name: "Bronze", discount: "-30%", tfts: 400 },
+      { name: "Silver", discount: "-40%", tfts: 600 },
+      { name: "Gold", discount: "-60%", tfts: 1000 },
```
Comment (AhmedHanafy725) [inline packages/playground/src/components/node_selector/TfNodeDetailsCard.vue:510]: The TFTs required to unlock the discount should be calculated based on the node resources.

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (AhmedHanafy725) [inline yarn.lock:1]: why do we have changes here? did we install a new package?

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (amiraabouhadid) [reply yarn.lock:1]: I did but then I removed them

**Comment by AhmedHanafy725** [comment]
- I think we should show the price after the discount on the node card
- the price approximation should be ceil not floor
- I think we need something to say what is that table on the top 
- As a user why can't I see the required TFTs if I am not logged in
image

**Comment by amiraabouhadid** [comment]
> * I think we should show the price after the discount on the node card
> * the price approximation should be ceil not floor
> * I think we need something to say what is that table on the top
> * As a user why can't I see the required TFTs if I am not logged in
>   image

good catch, fixed, thanks

**Review by MohamedElmdary — APPROVED** [review]

File: yarn.lock
Comment (AhmedHanafy725) [reply yarn.lock:1]: ok then the changes in yarn.lock should be reverted

**Comment by AhmedHanafy725** [comment]
sometimes the TFTs required is 0 and sometimes it has a value
image

Screenshot from 2024-07-10 17-19-49

**Review by samaradel — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**